akka-persistence-typed-tests/src/test/scala/akka/persistence/typed/ActiveActiveEventPublishingSpec.scala

```diff
@@ -145,6 +157,38 @@ class ActiveActiveEventPublishingSpec
       probe.expectMessage(Set("one", "two", "three"))
     }
 
+    "handle published events after replay" in {
```

Comment:
i think we should also handle a filtered message

Reply:
as i think that is broken as we don't start with the correct seenPerReplica

Reply:
Or a published before stopping would trigger the same I guess.